fili-core/src/main/java/com/yahoo/bard/webservice/web/handlers/workflow/DruidWorkflow.java

```diff
@@ -128,7 +134,7 @@ public DataRequestHandler buildWorkflow() {
         }
 
         // Requests sent to the NonUI we service are checked to see if they are too heavy to process
-        handler = new WeightCheckRequestHandler(handler, webService, weightUtil, mapper);
+        handler = new CacheWeightCheckRequestHandler(handler, webService, weightUtil, mapper, querySignedCacheService);
```

Comment:
I think we don't want to make this the default in DruidWorkflow. We want people to opt in on the new functionality.